docs/UserManuals/ConfigUI/GitHub.md

```diff
@@ -77,12 +84,17 @@ Each GitHub repo has at most ONE set of transformation rules.
 
 #### Additional Settings (Optional)
 
+![image-20221103121859001](https://user-images.githubusercontent.com/3294100/199653539-7fbbd76e-512f-480b-ba3c-066742ed3c4a.png)
+
+- PR Body Pattern: Extract the issue numbers closed by pull requests. The issue numbers are parsed from PR bodies that meet the following RegEx. For Example:
+   - ✅ Example 1: PR #321 body contains "**Closes #1234**" (PR #321 and issue #1234 will be mapped by the default RegEx)
+   - ❎ Example 2: PR #321 body contains "**Related to #1234**" (PR #321 and issue #1234 will NOT be mapped by the default RegEx)
+   - Default RegEx: `(?mi)(fix|close|resolve|fixes|closes|resolves|fixed|closed|resolved)[\s]*.*(((and )?(#|https:\/\/github.com\/%s\/%s\/issues\/)\d+[ ]*)+)`
+
 - Tags Limit: It'll compare the last N pairs of tags to get the "commit diff', "issue diff" between tags. N defaults to 10.
```

Comment:
The new version of this configuration will be like: ![image](https://user-images.githubusercontent.com/14050754/199666251-f0879ba6-7c7a-4474-be0f-1eb89e9bba56.png)

New Wording: 
Enable commits diff to calculate version-based metrics:

Calculate the commits diff between two consecutive tags that match the following RegEx. Issues closed by PRs which contain these commits will also be calculated. The result will be shown in table.refs_commits_diffs and table.refs_issues_diffs